zerver/lib/import_realm.py

```diff
@@ -1242,18 +1241,39 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
     bulk_import_model(data, CustomProfileFieldValue)
 
     # Import uploaded files and avatars
-    import_uploads(realm, os.path.join(import_dir, "avatars"), processes, processing_avatars=True)
-    import_uploads(realm, os.path.join(import_dir, "uploads"), processes)
+    import_uploads(
+        realm,
+        os.path.join(import_dir, "avatars"),
+        processes,
+        default_user_profile_id=first_user_profile.id if first_user_profile else None,
+        processing_avatars=True,
+    )
+    import_uploads(
+        realm,
+        os.path.join(import_dir, "uploads"),
+        processes,
+        default_user_profile_id=first_user_profile.id if first_user_profile else None,
+    )
```

Comment:
So, I would have expected uploads and avatars to not require this -- it might be better to pass `None` here (aka trigger an assertion failure). For avatars, the owner should always be the user whose avatar it is, and for and uploaded files, it'll be whoever sent the uploaded file in the service we're importing from.

Reply:
(Probably deserves a comment for why `default_user_profile_id=None` is explicitly passed in this case.

Reply:
Good point -- repushed.